pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -121,13 +121,14 @@ func volumeMounts(mi *miniov1.MinIOInstance) (mounts []corev1.VolumeMount) {
 		name = mi.Spec.VolumeClaimTemplate.Name
 	}
 
-	if mi.Spec.VolumesPerServer == 1 {
+	v, _ := strconv.Atoi(mi.Spec.VolumesPerServer)
```

Comment:
Same here, we should validate the string.

Reply:
validation is done initially here, https://github.com/minio/minio-operator/pull/169/files#diff-5fba95502c597c71c1ec5de3290ef01bR453 while someone submits a MinIO Tenant Yaml.